src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -330,7 +330,7 @@ describe('VAOS community care flow', () => {
     cy.findByText(/Choose provider/i).click();
     cy.findByText(/remove/i).click();
     cy.axeCheckBestPractice();
-    cy.findByText(/cancel/i).click();
+    cy.findByText(/cancel/i).click({ force: true });
```

Comment:
This was added due to an apparent bug with Cypress/Chrome. It is meant to be a quick fix while we [make a change to the `<va-modal>`'s CSS](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1113).